app/code/Magento/Inventory/Controller/Adminhtml/Stock/MassDelete.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Controller\Adminhtml\Stock;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Inventory\Ui\Component\MassAction\Filter;
+use Magento\Framework\Exception\CouldNotDeleteException;
+use Magento\InventoryApi\Api\StockRepositoryInterface;
+
+/**
+ * Delete Controller
+ */
+class MassDelete extends Action
+{
+    /**
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'Magento_Inventory::stock';
```

Comment:
in both Delete and MassDelete we have a duplication of this constant, that means that it should be declared in another place. 
Because currently, such code violates DRY